File: Beer O Clock/beer.c
//Beer O Clock in C  :)

#include<stdlib.h>

void beer();

int main(){
    beer();
}

void beer(){
    char ip;

    printf("Should I check if it is time to have Beer ? (Y / N) ");
    scanf("%c",&ip);

    if(ip == 'N' || ip == 'n') printf("We check later");

    else if (ip == 'Y' || ip == 'y') printf("It always is beer of clock. Cheers !!!");

    else printf("Invalid input, try again.");
};

File: Beer O Clock/beer.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout
});

readline.question(
  `Should I check if it is time to have Beer for Hacktoberfest? (Y / N): `,
  result => {
    result = result.toString().toLowerCase();
    if (result == "y") {
      console.log("It always is beer of clock. Cheers !!!");
    } else if (result == "n") {
      console.log("We check later.");
    } else {
      console.log("Invalid input, try again.");
    }
    readline.close();
  }
);
